docs/src/main/asciidoc/jms.adoc

```diff
@@ -19,26 +21,40 @@ To complete this guide, you need:
 * an IDE
 * JDK 1.8+ installed with `JAVA_HOME` configured appropriately
 * Apache Maven 3.5.3+
-* A running Artemis server, or Docker Compose to start one
-* GraalVM installed if you want to run in native mode.
+* A running Artemis server, or Docker to start one
+* GraalVM, or Docker, installed if you want to run in native mode.
```

Comment:
The two `or Docker` are weird. This should be reworded.

Reply:
Any recommendations? I couldn't really think of a better wording as they are really separate things. You dont need Docker for either requirement, but you could use it for one or both of them. Other guides seem to just ignore that you can use docker for native builds and so you dont need GraalVM installed.

Reply:
Ah no, it's OK, I somehow missed it was a bullet point.